hack/generator/pkg/codegen/pipeline_stage.go

```diff
@@ -59,6 +62,24 @@ func (stage PipelineStage) RequiresPrerequisiteStages(prerequisites ...string) P
 	return stage
 }
 
+// RequiresPostrequisiteStages declares which stages must be executed after this one has completed
+// This is not completely isomorphic with RequiresPrerequisiteStages as are supporting stages that are sometimes omitted
```

Comment:
Language here: `as are supporting stages`?

As there are?

Reply:
Revised the comment.